<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1007</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <widget class="QFrame" name="frame_ingreso">
      <property name="styleSheet">
       <string notr="true">QFrame {
    border: 1px solid #04b338;
}

QLabel {
    border: none;
}
</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>128</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="login_imageninstitucional">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap>../../imagenes/logo-altacalidad-uco.png</pixmap>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>138</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="login_lb_ingresousuario">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>40</height>
            </size>
           </property>
           <property name="font">
            <font>
             <family>Arial</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
           <property name="text">
            <string>INGRESO USUARIOS</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="login_in_usuario">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background-color: rgb(245, 245, 245);  /* Fondo gris claro para destacar en un fondo blanco */
    border-radius: 10px;
    font: 12pt &quot;Arial&quot;;
    border: 2px solid rgb(180, 180, 180);  /* Borde gris claro */
}

QLineEdit:hover {
    background-color: rgb(255, 255, 255);  /* Blanco puro cuando se pasa el mouse por encima */
    border-radius: 10px;
    font: 12pt &quot;Arial&quot;;
    border: 2px solid rgb(100, 149, 237);  /* Color azul claro en hover */
}

QLineEdit:focus {
    background-color: rgb(255, 255, 255);  /* Blanco puro cuando tiene el foco */
    border-radius: 10px;
    font: 12pt &quot;Arial&quot;;
    border: 2px solid rgb(0, 122, 204);    /* Color azul más oscuro cuando está en foco */
}
</string>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Usuario</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="login_in_contrasena">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    background-color: rgb(245, 245, 245);  /* Fondo gris claro para destacar en un fondo blanco */
    border-radius: 10px;
    font: 12pt &quot;Arial&quot;;
    border: 2px solid rgb(180, 180, 180);  /* Borde gris claro */
}

QLineEdit:hover {
    background-color: rgb(255, 255, 255);  /* Blanco puro cuando se pasa el mouse por encima */
    border-radius: 10px;
    font: 12pt &quot;Arial&quot;;
    border: 2px solid rgb(100, 149, 237);  /* Color azul claro en hover */
}

QLineEdit:focus {
    background-color: rgb(255, 255, 255);  /* Blanco puro cuando tiene el foco */
    border-radius: 10px;
    font: 12pt &quot;Arial&quot;;
    border: 2px solid rgb(0, 122, 204);    /* Color azul más oscuro cuando está en foco */
}

</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>20</number>
           </property>
           <property name="echoMode">
            <enum>QLineEdit::Password</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="placeholderText">
            <string>Contraseña</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="error_label">
         <property name="styleSheet">
          <string notr="true">QLabel#error_label{
	font: 17pt;
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>50</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="login_bt_ingresar">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
	border-top-left-radius: 0 px;
	border-bottom-left-radius: 30px;
	border-top-right-radius: 30px;
	border-bottom-right-radius: 0px
}
QPushButton:hover{
	border-top-left-radius: 0 px;
	border-bottom-left-radius: 30px;
	border-top-right-radius: 30px;
	border-bottom-right-radius: 0px
}


QPushButton {
    border-radius: 18px;
    background-color: qlineargradient(spread:reflect, x1:0, y1:0.532273, x2:1, y2:0.5, 
                                      stop:0.362832 rgba(0,128,0,255), /* Verde oscuro */
                                      stop:0.369469 rgba(0,128,0,255), 
                                      stop:1 rgba(0,255,0,255)); /* Verde claro */
}

QPushButton:hover {
    background-color: qlineargradient(spread:reflect, x1:0, y1:0.562273, x2:0.949, y2:0.545455, 
                                      stop:0 rgb(255,255,255), /* Cian oscuro */
                                      stop:1 rgb(255,255,255)); /* Cian claro */
}

QLabel {
    color: rgb(225, 225, 224);
    font: 12pt &quot;Arial&quot;;
}

</string>
           </property>
           <property name="text">
            <string>INGRESAR</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="login_label_linkuco">
           <property name="text">
            <string>www.uco.edu.co</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../imagenes/resources.qrc"/>
  <include location="../../imagenes/resources.qrc"/>
 </resources>
 <connections/>
</ui>
